### PR TITLE
Add error message guidelines section

### LINKED ISSUE #31: Add error message punctuation and formatting style guidelines.
Requested by Kate.

https://docs.google.com/document/d/1DscIXMmjBavQzBMUOHAsZ8UkY3Ildzqkve5s9g5mh58/edit#
For error message and message entries in the alphabetical terms listing:

The error messages in the following screenshot don’t have periods at the end because they’re in the middle of sentences (cyr note: I added periods to those), but it would be helpful to link out to additional instructions that specify punctuation for error messages. 

Another thing that might be helpful to specify is what punctuation should be used between the error code and the error message. For example, “Error code: 'FED2- 013'; Token expiration cannot exceed <maxTokenLifetime> seconds.” These types of messages appear at https://github.rackspace.com/ServiceAPIContracts/global-auth-keystone-extensions/blob/master/docs/specs/CID-1649.rst.

### PR DESCRIPTION
**Pull request by cyrichardson**
Fixes #31 

I'd created this section for the Helix documentation and thought it might work to respond to Kate's request.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-773676faf5/docs/style-guide/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-db256d082b/docs/style-guide/).

**Comment by cyrichardson** [comment]
@RackerKate Please review when you have time. Thanks!

File: style-guide/error-message-guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+       information, see :ref:`capitalization`.
+     - The requested image `$UUID` has automatic disk resizing disabled.
+     - THE REQUESTED IMAGE $UUID HAS AUTOMATIC DISK RESIZING DISABLED.
+   * - When possible, include a recommendation, either a potential fix or a
+       reference to a document for more information.
+
+       Messages should provide specific information about how the user should continue.
+     - The system is out of virtual IP addresses. Contact Support so they can
+       allocate more virtual IP addresses.
+       addresses.
```
[the remark this answers is not in the document]
Comment (cyrichardson) [reply style-guide/error-message-guidelines.rst]: fixed

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-3fe90dc896/docs/style-guide/)

**Pull request merged**